main.go

```diff
@@ -14,8 +14,10 @@
 package main
 
 import (
+	"compress/gzip"
 	"encoding/json"
 	"fmt"
+	"gopkg.in/alecthomas/kingpin.v2"
```

Comment:
cosmetic: this shouldn't be in the middle of all the standard library imports, I'll fix that up after merge.